js/main.js

```diff
@@ -109,3 +150,30 @@ function changeViewNav(event) {
 }
 
 $navBar.addEventListener('click', changeViewNav);
+
+$entryList.addEventListener('click', editEntry);
+
+var closestElement;
+var dataEntryIDValue;
+
+var $formHead = document.querySelector('h2');
+var dataEntriesIndex;
+
+function editEntry(event) {
+  if (event.target.matches('i')) {
+    closestElement = event.target.closest('li');
+    dataEntryIDValue = Number(closestElement.getAttribute('data-entry-id'));
+    for (i = 0; i < data.entries.length; i++) {
```

Comment:
you should definitely declare your variable here instead of using a global.

Reply:
Done.